queries.sql

```diff
@@ -1,11 +1,60 @@
-
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * FROM animals WHERE name LIKE '%mon';
+
 SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
-SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+SELECT name FROM animals WHERE neutered = True AND escape_attempts < 3;
+
 SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = true; 
-SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT * FROM animals WHERE neutered = True;
+
+SELECT * FROM animals WHERE name != 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--Begin animal transaction
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+
+-- another animal transaction
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+
+
+-- delete transaction
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+-- another delete transaction
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT SAVEPOINT_1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT_1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+
```

Comment:
- [ ] `README.md` file: I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same comment in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**
![Screenshot from 2023-06-13 18-19-27](https://github.com/harshi0102/vet-clinic-project/assets/83042742/ea27e2af-23fe-4ede-be8e-92ab3e8f6130)